src/components/Game/Game.js

```diff
@@ -1,65 +1,124 @@
-import React, { useContext } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
-import Button from "../Button";
 import "./Game.css";
-import { GameContext } from "../../contexts/gameContext";
+import Avatar from "../Avatar";
+import Button from "../Button";
+import PlayerCard from "../PlayerCard/PlayerCard";
 import ScoreInput from "../ScoreInput/ScoreInput";
+import Status from "../Status";
 import Swords from "../../images/swords.svg";
+import { GameContext } from "../../contexts/gameContext";
 import { Player } from "../../utils/player";
 
 const Game = props => {
   const game = useContext(GameContext);
   const [players, setPlayers] = game.usePlayers;
+  const [lowestPlayerLevel, setLowestPlayerLevel] = useState();
+  const [highestPlayerLevel, setHightestPlayerLevel] = useState();
+
+  useEffect(() => {
+    const levels = players.map(player => player.level);
+    const max = Math.max(...levels);
+    const min = Math.min(...levels);
+    const minLevelLength = levels.filter(level => min === level).length;
+    setLowestPlayerLevel(minLevelLength === 1 ? min : null);
+    setHightestPlayerLevel(max);
+  }, [players]);
 
-  const updatePlayerScore = (newScore, playerIndex) => {
+  const updatePlayer = (playerId, field, value) => {
     const newData = [...players];
-    newData[playerIndex].score = newScore;
+
+    const playerIndex = players.findIndex(p => {
+      return p.id === playerId;
+    });
+
+    newData[playerIndex][field] = value;
+
     setPlayers(newData);
   };
 
   const resetGame = () => {
     setPlayers(
       Array(2)
         .fill(null)
-        .map(() => new Player())
+        .map(() => new Player({}))
```

Comment:
Nit: handle no argument being passed to `new Player()` within the player definition, so that an empty object is not necessary.